ios/ci_scripts/ci_post_clone.sh

```
@@ -0,0 +1,41 @@
+# else
+#   cp .env.stag .env
+# fi
+
+# Install Flutter dependencies.
+flutter pub get
+
+# configure flutter fire
+# dart pub global activate flutterfire_cli
+
+# Install CocoaPods using Homebrew.
+HOMEBREW_NO_AUTO_UPDATE=1 # disable homebrew's automatic updates.
```

Shellcheck (0.10.0)

[warning] HOMEBREW_NO_AUTO_UPDATE appears unused. Verify use (or export if used externally).

(SC2034)